README.md

```diff
@@ -6,41 +6,27 @@ The following presents two alternatives for configuring your Kafka clients to us
 
 The first alternative is suited for Java clients where you have the ability to modify the client classpath to include the authentication libraries.
 
-The second alternative works with other clients beyond Java, it requires you running a local authentication server.
+The second alternative offers a solution for non-Java Kafka clients, but requires you to set up a local authentication server. This server's role is to securely exchange your application's default credentials with the Kafka client, enabling authentication and authorization for accessing the Kafka cluster.
+
+In either case, your client will use the Google Auth libraries in order to authenticate using the default environment credentials. By default on GCP environments such as GKE or GCE, it means the library will use the environment default credentials. This behavior can be modified to point to different credentials via the GOOGLE_APPLICATION_CREDENTIALS environment variable as described in [this article](https://github.com/googleapis/google-auth-library-java?tab=readme-ov-file#getting-application-default-credentials).
+
 
 ## Kafka Java Auth Client Handler
 
 Inside kafka-java-auth, you'll find an implementation of Kafka's [AuthenticateCallbackHandler](https://kafka.apache.org/20/javadoc/org/apache/kafka/common/security/auth/AuthenticateCallbackHandler.html) that is suited to have your Kafka clients authenticate with Google Cloud Managed Service for Apache Kafka clusters.
 
-Follow the steps below to get your client setup.
+Follow these two steps below to get your client setup.
 
-1. Build the supporting JAR.
-```
-cd kafka-java-auth
-mvn install
-```
+1. The library is available on Maven. Find instructions on how to add it to your project, tailored to your specific build system, [here](https://central.sonatype.com/artifact/com.google.cloud.hosted.kafka/managed-kafka-auth-login-handler).
 
-The relevant artifact will be generated and placed in your local maven repository. 
-
-2. Then you can include the following dependency in your build to include the library along with its dependencies.
-```
-<dependency>
-    <groupId>com.google.cloud.hosted.kafka</groupId>
-    <artifactId>managed-kafka-auth-login-handler</artifactId>
-    <version>1.0.0</version>
-</dependency>
-```
-
-3. Configure your Kafka client, including the following authentication properties.
+3. Configure your Kafka client, including the following client authentication properties.
```

Comment:
Replace 3. with 2.

Reply:
Done

Reply:
Perhaps the changes were not pushed yet?